rust/Dockerfile.sccache.aarch64-unknown-linux-gnu

```diff
@@ -5,7 +5,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 # RUN /sccache.sh aarch64-unknown-linux-gnu
 
 COPY sccache-prebuilt.sh /sccache.sh
-RUN /sccache.sh x86_64-unknown-linux-musl
+RUN /sccache.sh aarch64-unknown-linux-musl
```

Comment:
I think this needs to be x86_64 because sccache will run on x86_64, since it's cross compiling?